### DISCUSSION TITLE
Automatically download the very first 25 scores that someone sets.

### DISCUSSION THREAD

**Discussion by ziv-vy** [ppy/osu#30487] [Ideas]
Related issue: [#1999013](https://osu.ppy.sh/community/forums/topics/1999013?n=1)

When you first download osu!lazer, automatically download the first scores that said user sets.
The game may only download up to 3 scores on any difficulty, and up to 5 for any beatmap set, so that the user doesn't just save 25 scores of Triangles.
Right now, most old scores are only saved on the servers, and will probably be wiped after being replaced with a much better one. The chance for the user to just download the score for no reason is low.
In the future, it will be a fun journey to check out those replays (and see how bad you were).

**Comment by shinmorisawa** [comment]
Maybe this would be a good idea, but idk about space. space could be a big problem.

**Reply by ziv-vy** [reply]
I don't think that filesize is an issue at all. 
A replay for a 2* 2:30 min map is like ~30KB.
With this rate, 25 maps aren't even a Megabyte.
For comparison, osu!lazer itself has like 670MB

**Reply by ziv-vy** [reply]
Plus, you can just save it on the servers, but that's getting into the territory of osu-web and this is an osu-lazer discussion.
Actually... https://osu.ppy.sh/community/forums/topics/1999013?n=1
